docs/se/guides/tracing.adoc

```diff
@@ -163,30 +164,28 @@ WebServer server = WebServer.builder(createRouting(config))
 [source,java]
 .Update the `GreetService` class; 1) Add a new import and 2) Replace the `getDefaultMessageHandler` method:
```

Comment:
I missed this earlier. We no longer add a new import; we just replace the method.